packages/client/hmi-client/src/components/workflow/ops/funman/tera-funman-drilldown.vue

```diff
@@ -301,6 +312,27 @@
 							@remove="removeChartSetting"
 						/>
 						<hr />
+						<h5>Observable</h5>
+						<tera-chart-control
+							class="w-full"
+							:chart-config="{
+								selectedRun: 'fixme',
```

Comment:
Could you please add a comment about this `fixme`.

Reply:
The "fixme" looks like a copy-n-paste, it is sort of a legacy construct from when we were supporting multiple number of runs per output. The fixmes were there when we were transitioning out of the multi-run pattern - if there is time we should remove it.

Reply:
Move to different task